src/containerapp/azext_containerapp/_params.py

```diff
@@ -134,7 +134,10 @@ def load_arguments(self, _):
 
     # Springboard
     with self.argument_context('containerapp create', arg_group='Service Binding') as c:
```

Comment:
Recommend adding a example in _help.py for the new scenario.

Reply:
will add in the next commit